src/containers/Accounts/AccountHeader/styles.scss

```diff
@@ -3,10 +3,44 @@
 .account-header {
   margin-bottom: 16px;
 
+  .title {
+    padding-bottom: 4px;
+    color: $black-40;
+    font-size: 14px;
+    text-transform: uppercase;
+    @include semibold;
+  }
+
   .box-header {
     margin-top: 106px;
     margin-bottom: 80px;
-    text-align: left;
+
+    .title {
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+    }
+
+    h1 {
+      overflow: hidden;
+      text-overflow: ellipsis;
+    }
+
+    .door-account {
```

Comment:
This has a lot of shared styles from the purple "NFT" badge.  Really the only difference should be color.  Something like this is often called a `.badge` or `.pill`
https://livenet.xrpl.org/nft/00092710F86687692510076EFAD3F5C0416552A25801D8F410E83C9C00006201